app/models/studying/stage_of_degree.rb

```diff
@@ -0,0 +1,17 @@
+module Studying
+  class StageOfDegree < Base
+    attribute :degree_stage
+
+    validates :degree_stage, types: { method: :get_qualification_degree_status, message: "You must select an option" }
+    OPTIONS = {
+      final_year: "222750001",
+      second_year: "222750002",
+      first_year: "222750003",
+      other: "222750005",
```

Comment:
Is 'Other' a selectable option in the UI? If so I may need to tweak the API

Reply:
Yes, we have: `<%= f.govuk_radio_button :degree_stage, '222750005', label: { text: 'Other' } %>`

Reply:
changed to: `<%= f.govuk_radio_button :degree_stage, Studying::StageOfDegree::options["Other"], label: { text: 'Other' } %`